<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1363</width>
    <height>970</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_view_camera">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>901</width>
     <height>901</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_info">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>330</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Info:</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>30</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="layout_histogram"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
